### PR TITLE
Feat backend init

### PR DESCRIPTION
**Pull request by SimonPesut**
Added .net backend to Project

### CODE REVIEW

File: .gitignore
Comment (Ascyt) [inline .gitignore:1]: Put this in the backend/ folder, and make sure it works. You can have multiple .gitignores in a repo

File: .gitignore [hunk 1]
```diff
@@ -36,3 +36,37 @@ Temporary Items
...
+# Temporary files
+*.tmp
+
+# Database
+*.db
+*.db-shm
+*.db-wal
+
+# Processed files
+*.vsidx
```
Comment (Ascyt) [inline .gitignore:72]: Also maybe add the directories these files are stored in

File: .gitignore
Comment (Ascyt) [reply .gitignore:1]: image
Add the full .vs folder to .gitignore

**Review by Ascyt — CHANGES_REQUESTED** [review]
Update .gitignore, add full directories into there, for example you added a lot of files of `.vs`

**Comment by SimonPesut** [comment]
Changed your code review

**Review by Ascyt — APPROVED** [review]

**Pull request merged**